eth/backend.go

```diff
@@ -483,6 +483,17 @@ func New(ctx context.Context, stack *node.Node, config *ethconfig.Config, logger
 						logItems = append(logItems, eth.ProtocolToString[protocol], strconv.Itoa(count))
 					}
 					logger.Info("[p2p] GoodPeers", logItems...)
+
+					if err := chainKv.Update(context.Background(), func(tx kv.RwTx) error {
+						if err = stagedsync.UpdateMetrics(tx); err != nil {
+							return err
+						}
+
+						return nil
+					}); err != nil {
+						logger.Warn("Could not update metrics", "err", err)
```

Comment:
each log line in erigon has prefix: like `[downloader]`, please add, otherwise users/devs hard to understand where it happening.